src/foam/nanos/boot/DAOConfigSummaryView.js

```diff
@@ -162,28 +186,81 @@
          }, x);
        });
 
+       var updateSections = [];
+       var i = 0;
+
        this.filteredDAO.select().then(function(specs) {
          specs.array.sort(function(o1, o2) { return foam.String.compare(o1.id.toUpperCase(), o2.id.toUpperCase())}).forEach(function(spec) {
            var label = foam.String.capitalize(spec.id.substring(0, spec.id.length-3));
            var l     = label.charAt(0);
 
            if ( l != currentLetter ) {
+            var lSection;
+             var showSection = foam.core.SimpleSlot.create({value: true});
+             i = updateSections.length;
+
+             var updateSlot = foam.core.SimpleSlot.create({value: false});
+             updateSections.push(updateSlot);
+
+             updateSections[i].sub(function() {
+               var doNeedToShow = false;
+               //first child is a header
+               for (var j = 1; j <  lSection.instance_.childNodes.length; j++) {
+                if ( lSection.instance_.childNodes[j].shown ) {
```

Comment:
Why are you calling instance_.childNodes[] instead of just .childNodes?

Reply:
It's not accessible for .childNodes

Reply:
Are you sure?

Reply:
Not really,
but I'm getting exception otherwise